src/Data/Food/Retail.elm

```diff
@@ -186,24 +185,34 @@ elecImpact elecNeeds volume =
             )
 
 
-transportImpact : Length -> Mass -> Process -> Impacts
-transportImpact distance mass =
-    .impacts
-        >> Impact.mapImpacts
-            (\_ impact ->
-                impact
-                    |> Unit.impactToFloat
-                    |> (*) (Length.inKilometers distance * Mass.inMetricTons mass)
-                    |> Unit.impact
-            )
+transportImpact : Db -> Mass -> Distribution -> WellKnown -> Transport
+transportImpact db mass (Distribution _ needs) wellKnown =
```

Comment:
Nit: as this now strictly depends on `Distribution`, we should probably rename this function to `distributionTransportImpact`